model/src/test/test_ar_process.py

```diff
@@ -11,33 +11,44 @@ def test_ar_can_be_sampled():
     Check that an AR process
     can be initialized and sampled from
     """
-    ar1 = ARProcess("arprocess", 5, jnp.array([0.95]), jnp.array([0.5]))
+    ar1 = ARProcess("ar1process")
     with numpyro.handlers.seed(rng_seed=62):
-        # can sample with and without inits
-        ar1(duration=3532, inits=jnp.array([50.0]))
-        ar1(duration=5023)
+        # can sample
+        ar1(
+            n=3532,
+            init_vals=jnp.array([50.0]),
+            autoreg=jnp.array([0.95]),
+            noise_sd=jnp.array([0.5]),
```

Comment:
I think this could just be  `noise_sd=0.5,` throughout?

Reply:
Fixed in e21cce6